data/pycon-au-2015/videos/lightning-talks-closing.json

```diff
@@ -1,24 +1,24 @@
 {
-  "id": 3986, 
-  "category": "PyCon AU 2015", 
-  "slug": "lightning-talks-closing", 
-  "title": "Lightning Talks & Closing", 
-  "summary": "", 
-  "description": "00:00 Intro\n00:25 Nic Couglan - Asyncio and native co-routines in python 3.5\n05:35 Tyson Clugg - Selenium Page Adaptor\n10:25 Tom Eastman - Serialisation formats are not toys\n16:00 Ben Davies - My Eyes, the goggles do nothing \n20:43 Paula Andrea Martinez - Software Carpentry\n24:17 Alex Hill - Django Many to Many Minus the Middle Men\n29:34 Adam Breniki - Who remembers VisiCalc\n33:34 Fraser Tweedale - Fast and Loose Reasoning and Parametricity\n38:47 Amber 'Hawkie' Brown - Pythonic Naming\n42:20 Sam Bishop - Distraction\n47:36 Tim Bell - Time Travel Debugger\n52:29 Brian Thron - Roll your own crypto\n57:18 Thuran Sunti - UQ Robotics Club\n1:02:35 Robert Collins - Data driven testing\n\n", 
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "creativeCommon", 
-  "thumbnail_url": "https://i.ytimg.com/vi/_pfJZfdwkgI/hqdefault.jpg", 
-  "duration": null, 
+  "id": 3986,
+  "category": "PyCon AU 2015",
+  "slug": "lightning-talks-closing",
+  "title": "Lightning Talks & Closing",
+  "summary": "",
+  "description": "00:00 Intro 00:25 Nic Couglan - Asyncio and native co-routines in python\n3.5 05:35 Tyson Clugg - Selenium Page Adaptor 10:25 Tom Eastman -\nSerialisation formats are not toys 16:00 Ben Davies - My Eyes, the\ngoggles do nothing 20:43 Paula Andrea Martinez - Software Carpentry\n24:17 Alex Hill - Django Many to Many Minus the Middle Men 29:34 Adam\nBreniki - Who remembers VisiCalc 33:34 Fraser Tweedale - Fast and Loose\nReasoning and Parametricity 38:47 Amber 'Hawkie' Brown - Pythonic Naming\n42:20 Sam Bishop - Distraction 47:36 Tim Bell - Time Travel Debugger\n52:29 Brian Thron - Roll your own crypto 57:18 Thuran Sunti - UQ\nRobotics Club 1:02:35 Robert Collins - Data driven testing\n",
```

Comment:
Newlines are lost. Same as for `data/pycon-au-2015/videos/lightning-talks-19.json`